tests/orchestrator/test_entity.py

```diff
@@ -23,6 +23,14 @@ def generator_function_call_entity(context):
     outputs.append(x)
     return outputs
 
+def generator_function_catch_entity_exception(context):
+    entityId = df.EntityId("Counter", "myCounter")
+    try:
+        yield context.call_entity(entityId, "add", 3)
+        return "Failure"
```

Comment:
Personally, it is a bit confusing to have the "not exception thrown" path return "failure" and the exception thrown path return "Success". I understand that this is because these test cases are trying to prove that an exception is thrown, but it may be more clear to just change the strings to "No exception thrown" and "Exception thrown".